node-registrar/pkg/db/nodes.go

```diff
@@ -23,6 +23,28 @@ func (db *Database) ListNodes(filter NodeFilter, limit Limit) (nodes []Node, err
 		query = query.Where("twin_id = ?", *filter.TwinID)
 	}
 
+	// Filter by online status (node sent an uptime report in the last 30 minutes)
+	if filter.Online != nil {
+		// Calculate the cutoff time (30 minutes ago by default)
+		cutoffMinutes := int64(30) // Default to 30 minutes
```

Comment:
the nodes sends the uptime report every 40 minutes, is that cutoff time related to that? if so it should be 40, also can we define the default as a global const we can reuse, or re adjust if we decided to update the interval of the uptime reports?

Reply:
I wanted to ask @xmonader about this, but I forgot. I will adjust it since you confirmed that no changes happened to the report intervals from the ZOS side.